Polygon_mesh_processing/examples/Polygon_mesh_processing/detect_features_example.cpp

```diff
@@ -0,0 +1,48 @@
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+#include <CGAL/Surface_mesh.h>
+
+#include <CGAL/Polygon_mesh_processing/Detect_features_in_polygon_mesh.h>
+#include <CGAL/Mesh_3/properties_Surface_mesh.h>
+
+#include <fstream>
+
+typedef CGAL::Exact_predicates_inexact_constructions_kernel K;
+typedef CGAL::Surface_mesh<K::Point_3>             Mesh;
+typedef boost::graph_traits<Mesh>::face_descriptor face_descriptor;
+
+namespace PMP = CGAL::Polygon_mesh_processing;
+
+int main(int argc, char* argv[])
+{
+  const char* filename = (argc > 1) ? argv[1] : "data/P.off";
+  std::ifstream input(filename);
+
+  Mesh mesh;
+  if (!input || !(input >> mesh))
+  {
+    std::cerr << "Not a valid input file." << std::endl;
+    return 1;
+  }
+
+
+  typedef typename boost::property_map<Mesh,CGAL::face_patch_id_t<int> >::type PatchID;
+  PatchID pid = get(CGAL::face_patch_id_t<int>(), mesh);
+  typedef typename boost::property_map<Mesh,CGAL::vertex_incident_patches_t<int> >::type VIP;
```

Comment:
Actually it should be VIMap, the vertex index is the property. In general when the property type is known we name it, replacing the property adjective.